casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -202,13 +201,13 @@ class RuntimeManagerImpl[F[_]: Concurrent] private[rholang] (
 
   private def deployPaymentSource(amount: Long, name: String = "__SCALA__"): String =
     s"""
-       | new rl(`rho:registry:lookup`), poSCh in {
-       |   rl!(`rho:rchain:pos`, *poSCh) |
-       |   for(@(_, PoS) <- poSCh) {
-       |     @PoS!("pay", $amount, "$name")
-       |   }
-       | }
-       """.stripMargin
+       # new rl(`rho:registry:lookup`), poSCh in {
+       #   rl!(`rho:rchain:pos`, *poSCh) |
+       #   for(@(_, PoS) <- poSCh) {
+       #     @PoS!("pay", $amount, "$name")
+       #   }
+       # }
+       """.stripMargin('#')
```

Comment:
really, this should've been a separate commit. It's even a different file, I don't understand why you didn't split this out.